fre/make/create_checkout_script.py

```diff
@@ -12,8 +12,30 @@
 fre_logger = logging.getLogger(__name__)
 
 def checkout_create(yamlfile, platform, target, no_parallel_checkout, jobs, execute, verbose):
-    """
-    Create the checkout script for bare-metal or container build
+    """Create the checkout script for bare-metal or container build
+
+    :param yamlfile: Model compile YAML file
+    :type yamlfile: str
+    :param platform: FRE platform
+    :type platform: str
+    :param target: Predefined FRE targets
+    :type target: str
+    :param no_parallel_checkout: Option to turn off parallel checkouts
+    :type no_parallel_checkout: bool
+    :param jobs: Number of jobs to run simultaneously
+    :type jobs: int
+    :param execute: Use this to run the created checkout script
```

Comment:
It's not clear what `this` is referring to...
Does this function also execute the checkout script?  That should be added at the top of the docstring